### PR TITLE
Add appstream metainfo for EknServicesMultiplexer

### PR DESCRIPTION
**Pull request by dylanmccall**
https://phabricator.endlessm.com/T29339

### CODE REVIEW

File: com.endlessm.BaseEknServicesMultiplexer.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>com.endlessm.BaseEknServicesMultiplexer</id>
+  <metadata_license>CC0</metadata_license>
+  <project_license>LGPL-2.1+</project_license>
+  <name>Endless Desktop Search</name>
+  <summary>Desktop search provider for Endless applications</summary>
```
Comment (dbnicholson) [inline com.endlessm.BaseEknServicesMultiplexer.metainfo.xml]: So this doesn't appear to duplicate the real `EknServicesMultiplexer` in flatpak, maybe use a slightly different name and summary here? Maybe like "Extension points for Endless Desktop Search"?

**Comment by dbnicholson** [comment]
bot, retest this please

**Comment by dbnicholson** [comment]
I pushed https://github.com/endlessm/eos-build/commit/7963479bf49780b4cce5fff5de99db846743ee72 to fix the build error.

**Comment by dylanmccall** [comment]
Yeah, I feel a little odd about that, but indeed it felt more accurate to me from the perspective of something like gnome-software :)

**Pull request merged**